packages/sw-background-sync-queue/src/lib/queue-utils.js

```diff
@@ -0,0 +1,94 @@
+import {getDb} from './background-sync-idb-helper';
+
+/**
+ * takes a request and gives back JSON object that is storable in IDB
+ *
+ * @param {Request} request request object to transform
+ * into iDB storable object
+ * @param {Object} config config object to be
+ * stored along in the iDB
+ * @return {Object} indexable object for iDB
+ *
+ * @memberOf RequestManager
+ */
+async function getQueueableRequest(request, config) {
```

Comment:
Can you switch to using an anonymous object with named properties instead of position parameters?